mixins/localize/test/localize-mixin.test.js

```diff
@@ -617,4 +625,18 @@ describe('LocalizeMixin', () => {
 
 	});
 
+	describe('common', () => {
+
+		let elem;
+		beforeEach(async() => {
+			elem = await fixture(`<${localizeCommonTag}></${localizeCommonTag}>`);
+		});
+
+		it('should localize "&"', () => {
+			const localized = elem.localizeCharacter('&');
+			expect(localized).to.equal('ampersand');
```

Comment:
Mostly just a smoke test as I didn't want to repeat everything from `@brightspace-ui/intl`.